source/op/prod_env_mat_multi_device.cc

```diff
@@ -25,6 +25,26 @@ REGISTER_OP("ProdEnvMatA")
     .Output("nlist: int32");
     // only sel_a and rcut_r uesd.
 
+// an alias of ProdEnvMatA -- Compatible with v1.3
+REGISTER_OP("DescrptSeA")
```

Comment:
Do we need to register `DescrptSeA`? the op name has changed to `ProdEnvMatA` by `convert`

Reply:
We need to load the old graph first when executing `convert`.

Reply:
I see